app/src/main/kotlin/co/netguru/baby/monitor/client/feature/settings/ClientSettingsFragment.kt

```diff
@@ -13,7 +12,6 @@ import co.netguru.baby.monitor.client.BuildConfig
 import co.netguru.baby.monitor.client.R
 import co.netguru.baby.monitor.client.common.base.BaseDaggerFragment
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/59ecd15255bdd529a391ea0d5072dc37d88e9138/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/settings/ClientSettingsFragment.kt#L13"></span>co.netguru.baby.monitor.client.common.extensions.* is a wildcard import. Replace it with fully qualified imports.</td>
    </tr>
  </tbody>
</table>